src/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface SearchedPatient {
+  patientId: number;
+  uuid: string;
+  identifiers: Array<Identifier>;
+  person: {
+    age: number;
+    birthdate: string;
+    display: string;
+    gender: string;
+  };
+  display: string;
+}
+
+export type Identifier = {
```

Comment:
Its strange that the upper one (`SearchedPatient`) is an interface, while this was defined as a `type`. Maybe make both `interface`s as this is what they are.

Reply:
Fixed